gradle/libs.versions.toml

```
@@ -1,32 +1,28 @@
 [versions]
-loom = "1.3.+"
-loom_vineflower = "1.11.+"
+loom = "1.7.+"
 
 minotaur = "2.7.+"
 cursegradle = "2.+"
 github_release = "2.+"
 machete = "2.+"
 
-minecraft = "1.20.2"
-quilt_mappings = "2"
-fabric_loader = "0.14.22"
-fabric_api = "0.89.1+1.20.2"
-yacl = "3.2.1+1.20.2"
-mod_menu = "8.0.0"
-mixin_extras = "0.2.0-rc.4"
+minecraft = "1.21"
+quilt_mappings = "1"
+fabric_loader = "0.15.11"
+fabric_api = "0.100.1+1.21"
```

```suggestion
fabric_api = "0.100.2+1.21"
```
I would have bumped one time as 0.100.1 was for MC 1.21-rc1 as we can see in the GitHub releases https://github.com/FabricMC/fabric/releases

---

Given that said Fabric API version is marked on MR as 1.21 compatible I don't think this is needed, the RCs are usually a near perfect match to the final release anyways.